modules/storage-persistent-storage-nfs-export-settings.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-nfs.adoc
+
+= Export Settings
+
+In order to enable arbitrary container users to read and write the volume, 
+each exported volume on the NFS server should conform to the following 
+conditions:
+
+* Every export must be exported using the following format:
++
+----
+/<example_fs> *(rw,root_squash)
+----
+
+* The firewall must be configured to allow traffic to the mount point.
+** For NFSv4, configure the default port `2049` (*nfs*) and port 
+`111` (*portmapper*).
++
+.NFSv4
+----
+# iptables -I INPUT 1 -p tcp --dport 2049 -j ACCEPT
+# iptables -I INPUT 1 -p tcp --dport 111 -j ACCEPT
+----
+
+** For NFSv3, there are three ports to configure:
+`2049` (*nfs*), `20048` (*mountd*), and `111` (*portmapper*).
++
+.NFSv3
+----
+# iptables -I INPUT 1 -p tcp --dport 2049 -j ACCEPT
+# iptables -I INPUT 1 -p tcp --dport 20048 -j ACCEPT
+# iptables -I INPUT 1 -p tcp --dport 111 -j ACCEPT
+----
+
+* The NFS export and directory must be set up so that it is accessible by 
```

Comment:
I've adjusted this. I didn't consider it previously when converting the 3.x content, because mentally I considered them one entity due to the one line in the `exports` file that represents the NFS export.